src/Symfony/Component/Console/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+7.2
+---
+
+ * Add the `definition` option to configure arguments and options in the `AsCommand` attribute
```

Comment:
definitionS ?

Reply:
I renamed `definition` to `inputDefinition`

as we call `public function setDefinition(array|InputDefinition $definition): static`, keep singular seems more consistent